test/test_monge_ampere.py

```diff
@@ -14,13 +14,24 @@ def fix_boundary(request):
     return request.param
 
 
-def test_uniform_monitor(method, exports=False):
+@pytest.fixture(params=[2, 3])
+def mesh_dim(request):
+    if request.param == 2:
+        def mesh(n):
+            return UnitSquareMesh(n, n)
+    else:
+        def mesh(n):
+            return UnitCubeMesh(n, n, n)
+    return mesh
```

Comment:
The name `mesh_dim` is a bit misleading, given that the result is a function that returns a mesh. Elsewhere we've had a fixture for the dim and then a function like `uniform_mesh` that maps it to the desired initial mesh.

Reply:
good idea - much cleaner